contract-examples/contracts/ExampleTxAllowList.sol

```diff
@@ -1,14 +1,195 @@
 //SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.5;
```

Comment:
why is this changed? could we either stay on 0.8.0 or upgrade to 0.8.5 version for all tests?

Reply:
Good catch, will go back to 0.8.0, 0.8.5 produces warnings and I have since removed the 0.8.5 feature I was using. We might need to upgrade to better deal with errors in the future (when our precompiles return better errors), but we don't need to do that now.